app/views/covid19_mailer/covid_19_delay.text.erb

```diff
@@ -0,0 +1,7 @@
+Dear <%= @application_form.first_name || 'candidate' %>
+
+Given the impact of coronavirus (COVID-19) on schools and universities, your teacher training application might be processed more slowly than usual.
+
+To check the status of your application, sign in to Apply for teacher training https://www.apply-for-teacher-training.education.gov.uk/candidate/sign-in
```

Comment:
We have `ViewHelper#candidate_sign_in_url` for generating the sign in link for a candidate, see `app/helpers/view_helper.rb:54`.

```suggestion
To check the status of your application, sign in to <%= candidate_sign_in_url(candidate) %>
```

Reply:
Might have better not to introduce an expiring token into the email 

> How are we going to trigger this mailer? Command line?

Yep, was thinking just use the rails runner on an Azure shell.

Reply:
👍 , updated.